harness/determined/profiler.py

```diff
@@ -1053,3 +1053,37 @@ def measure(self, batch_idx: int) -> Dict[str, Measurement]:
         except Exception as e:
             logging.warning(f"{LOG_NAMESPACE}: error while measuring GPU memory: {e}")
             return {}
+
+
+class DummyProfilerAgent(ProfilerAgent):
+    def __init__(self) -> None:
+        pass
+
+    def __enter__(self) -> "DummyProfilerAgent":
+        pass
+
+    def __exit__(
+        self,
+        exc_type: Optional[Type[BaseException]],
+        exc_value: Optional[BaseException],
+        traceback: Optional[TracebackType],
```

Comment:
`*_: Any` should suffice here, rather than naming the arguments you are going to ignore